packageregistry/github.go

```diff
@@ -136,6 +139,10 @@ func (ga *githubPackageRegistryPackageDiscovery) GetPackage(packageName string)
 	return githubRegistryCreatePackageWrapper(repository, latestVersion, pkgVersions), nil
 }
 
+func (ga *githubPackageRegistryPackageDiscovery) GetPackageDownloadStats(packageName string) (DownloadStats, error) {
+	return DownloadStats{}, nil
```

Comment:
Return error please ie. download stats is not supported, otherwise it will cause bugs in client code assuming it to be `0` downloads